qemu/tests/lvm_with_read_only.py

```diff
@@ -0,0 +1,45 @@
+import time
+
+from avocado.utils import process
+
+from virttest import env_process
+
+
+def run(test, params, env):
+    """
+    Boot guest after disable ept/npt:
+    1) Create lvm with read-only permission
+    2) Boot up guest with lvm
+    3) Unplug the disk
+    4) Hotplug blockdev node with auto-read-only
+
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment
+    """
+
+    def create_lvm():
+        devname = process.getoutput(params["get_devname_command"])
+        pvcreate_command = (params["pvcreate_command"] % devname)
+        process.system(pvcreate_command)
+        vgcreate_command = (params["vgcreate_command"] % devname)
+        process.system(vgcreate_command)
+        process.system(params["lvcreate_command"])
```

Comment:
Could we call the avocado-vt framework interface `./virttest/lvm.py` to implement this part?